frontend/src/components/dashboard/common/services.ts

```diff
@@ -0,0 +1,37 @@
+import eam from '../../../assets/img/eam.svg';
+import bs from '../../../assets/img/binary-storage.svg';
+import sc from '../../../assets/img/server-computing.svg';
+import slc from '../../../assets/img/sl-computing.svg';
+import { AppRoute } from '../../../common/enums/app/app-route.enum';
+
+interface IService {
+  title: string;
+  img: string;
+  route: AppRoute;
+}
```

Comment:
```suggestion
type ServiceMenuItem {
  title: string;
  img: string;
  route: AppRoute;
}
```
 move to `frontend/src/common/types/ui`